app/main.py

```diff
@@ -1 +1,58 @@
-# write your code here
+class Animal:
+    """
+    Class animal contains animals name,
+    also health and hidden with default parameters.
+    It's store alive animals in alive list of dicts,
+    if animal's health > 0.
+    """
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f'kotName: {self.name}, Health: {self.health}, '
+                f'Hidden: {self.hidden}pies').replace(
+            "kot", "{").replace("pies", "}")
```

Comment:
The `__repr__` method has a formatting issue. The string replacement logic seems unnecessary and confusing. Consider directly using the format method or f-string without replacements, like `return f'{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}'`.